scripts/dev/templates/Dockerfile.python_formatting

```
@@ -0,0 +1,13 @@
+FROM {{base_image}}
+
+
+{% for add in adds %}
```

```suggestion
{% for file in files_to_add %}
```
looks a bit nicer I think!